### PR TITLE
Fix pulsar box sizing

### PR DESCRIPTION
**Pull request by akulkapoor**
Before:
screen shot 2018-05-09 at 2 33 24 pm

After:
screen shot 2018-05-09 at 2 35 41 pm

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit 084c7fcb02245518b59ef14081983672815c7976

https://deploy-preview-169--gestalt.netlify.com

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please add a [changelog.md](https://github.com/pinterest/gestalt/blob/master/CHANGELOG.md) entry for your changes.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: packages/gestalt/src/Pulsar/Pulsar.css [hunk 1]
```diff
@@ -4,6 +4,7 @@
   animation-name: PulseAnimation;
   border: 8px solid #0084ff;
   border-radius: 50%;
+  box-sizing: border-box;
```
Comment (chrislloyd) [inline packages/gestalt/src/Pulsar/Pulsar.css]: You should be able to compose this from layout to save some css size

**Comment by christianvuerings** [comment]
@akulkapoor Could you also add this to the `Changelog` file?

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Bundle size increase
  </td>
    </tr>
  </tbody>
</table>



Filename | Size Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip
--- | --- | --- | --- | --- | --- | --- |
gestalt.js | :small_red_triangle: 0.0% | :small_red_triangle: 0.0% | 282.03 KB | 282.04 KB | 63.06 KB | 63.06 KB
gestalt.es.js | :small_red_triangle: 0.0% | :small_red_triangle: 0.0% | 279.39 KB | 279.39 KB | 62.68 KB | 62.68 KB
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**